app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/BulkConfigurationTransfer.php

```diff
@@ -65,37 +65,35 @@ public function execute(
 
         $types = $this->getProductTypesBySkus->execute($skus);
 
-        foreach ($types as $sku => $type) {
-            if ($this->isSourceItemManagementAllowedForProductType->execute($type)) {
-                foreach ($skus as $sku) {
-                    $qry = $connection
-                        ->select()
-                        ->from($tableName, 'notify_stock_qty')
-                        ->where('sku = ?', $sku)
-                        ->where('source_code = ?', $originSource);
+        foreach ($skus as $sku) {
+            if ($this->isSourceItemManagementAllowedForProductType->execute($types[$sku])) {
+                $qry = $connection
```

Comment:
please provide meaningful names for variables.
Usage of shortening like `qry` and `res` is discouraged